lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -120,18 +120,6 @@
     HAS_BOTO = False
 
 
-def boto_exception(err):
```

Comment:
This will break, since there's no wildcard import here `boto_exception` will need to be included [here](https://github.com/leedm777/ansible/blob/e81669f4e5c14120a42c1b735b35493739bcfd9e/lib/ansible/modules/cloud/amazon/iam_cert.py#L111)